src/RegistryPageController.php

```diff
@@ -312,13 +311,23 @@ protected function queryList(array $where, array $orderby, $start, $pageLength,
 
         $tableName = DataObject::getSchema()->tableName($dataClass);
 
-        $resultColumns = $this->dataRecord->getDataSingleton()->summaryFields();
-        $resultColumns['ID'] = 'ID';
+        $summarisedModel = $this->dataRecord->getDataSingleton();
+        $resultColumns = $summarisedModel->summaryFields();
+
+        // Remove any non database residing summary fields before trying to erroneously SELECT them.
+        $resultDBOnlyColumns = [];
+        foreach ($resultColumns as $summaryFieldKey => $summaryFieldValue) {
+            if (!$summarisedModel->hasMethod("get{$summaryFieldKey}")) {
```

Comment:
`$summaryFieldKey` could legitimately be `getThumbnail`, so we'd need to check for `!...->hasMethod($summaryFieldKey) && !...->hasMethod("get{$summaryFieldKey}")` I think

Reply:
Hmm, a summary field should be a pointer to a field and not to a method. You want to follow the logic of core as closely as possible with this and I'm pretty sure they don't accept first class methods as values but the do accept pseudo properties (ie: properties populated by `get{name}`

Reply:
pseudo fields fall back to `__get`, which in turn relies on `hasField`:
https://github.com/silverstripe/silverstripe-framework/blob/4.0/src/ORM/DataObject.php#L2479

It seems only templates check properties _and_ methods. So this is ready for merge.